integration/query_command_compat_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+
+	"github.com/FerretDB/FerretDB/integration/setup"
+)
+
+// queryCountCompatTestCase describes command query compatibility test case.
+type queryCountCompatTestCase struct {
+	query   bson.D // optional
+	command bson.D // defaults to count collection
+}
+
+// testQueryCountCompat tests command query compatibility test cases.
+func testQueryCountCompat(t *testing.T, testCases map[string]queryCountCompatTestCase) {
```

Comment:
Hmm, if I'm not mistaken, `count_compat_test.go` has some similar things... Does it make sense to combine those two, so we don't have duplicate code?

Reply:
yes thanks for spotting this. I combined them